installer/lib/phx_new/generator.ex

```diff
@@ -83,6 +83,28 @@ defmodule Phx.New.Generator do
     end
   end
 
+  def prod_only_config_inject(path, file, to_inject) do
+    file = Path.join(path, file)
+
+    contents =
+      case File.read(file) do
```

Comment:
The file will always exist because we just generated it, so you can use `File.read!` here.

Reply:
`Mix.Tasks.Phx.New.Ecto.run/1` depends on the presence of this check in installer/test/phx_new_ecto_test.exs. Is that something that needs to change?

Reply:
Good call. Trust the tests, not me. So this is good to go. :)